test/unit-test/test_lib/test_analysis/test_network.py

```diff
@@ -306,25 +306,27 @@ def test_nick_receiver_from_conn_comp(self):
     @mock.patch('lib.analysis.network.message_number_graph', autospec=True)
     @mock.patch('lib.analysis.network.user.keywords', autospec=True)
     def test_identify_hubs_and_experts(self, mock_keywords, mock_msg_graph):
-    
-        top_hub_ = util.load_from_disk(current_directory+ "/data/top_hub")
-        top_keyword_overlap_ = util.load_from_disk(current_directory+ "/data/top_keyword_overlap")
-        top_auth_ = util.load_from_disk(current_directory+ "/data/top_auth")
-        message_graph = util.load_from_disk(current_directory+ "/data/test_hits_message_graph")
-        keyword_dict_list = util.load_from_disk(current_directory+ "/data/keyword_dict_list")
-        user_keyword_freq_dict = util.load_from_disk(current_directory+"/data/user_keyword_freq_dict")
-        user_words_dict_list = util.load_from_disk(current_directory+"/data/user_words_dict_list")
-        nicks_for_stop_words = util.load_from_disk(current_directory+"/data/nicks_for_stop_words")
-        keywords_for_channels = util.load_from_disk(current_directory+"/data/keywords_for_channels")
-        keywords_return = util.load_from_disk(current_directory+ "/data/keywords_return")
+        
+        log_data = util.load_from_disk(current_directory+ "/data/hits/log_data")
+        nicks = util.load_from_disk(current_directory+ "/data/hits/nicks")
+        nick_same_list = util.load_from_disk(current_directory+ "/data/hits/nick_same_list")
+        top_hub_ = util.load_from_disk(current_directory+ "/data/hits/top_hub")
+        top_keyword_overlap_ = util.load_from_disk(current_directory+ "/data/hits/top_keyword_overlap")
```

Comment:
@kaushik-rohit  Why do the variable names have underscores at the end?

Reply:
@prasadtalasila those are the expected result while top_hub would be generated result. I can change the convention if necessary.

Reply:
@kaushik-rohit Let's use a single convention across the project. Can you check the tests written by others and suggest one convention. All the tests can be changed to have that convention.